tests/foreman/cli/test_computeresource_vmware.py

```diff
@@ -149,3 +149,86 @@ def test_positive_provision_end_to_end(
     )
     host_info = sat.cli.Host.info({'id': host['id']})
     assert host_info['status']['build-status'] == 'Installed'
+
+
+@pytest.mark.e2e
+@pytest.mark.on_premises_provisioning
+@pytest.mark.parametrize('setting_update', ['destroy_vm_on_host_delete=True'], indirect=True)
+@pytest.mark.parametrize('vmware', ['vmware7', 'vmware8'], indirect=True)
+@pytest.mark.parametrize('pxe_loader', ['bios'], indirect=True)
+@pytest.mark.rhel_ver_match('[8]')
+@pytest.mark.tier3
+def test_positive_image_provision_end_to_end(
+    request,
+    setting_update,
+    module_provisioning_sat,
+    module_sca_manifest_org,
+    module_location,
+    pxe_loader,
+    module_vmware_cr,
+    module_vmware_image,
+    module_vmware_hostgroup,
+    vmware,
+    vmwareclient,
+):
+    """Provision a host with image on vmware compute resource with
+    the help of hostgroup.
+
+    :id: 8f0e2278-b897-4927-9c21-d84313623cc4
+
+    :steps:
+
+        1. Configure provisioning setup.
+        2. Create VMware CR
+        3. Create VMware image
+        4. Configure host group setup.
+        5. Provision a host on VMware
+        6. Verify created host on VMware with wrapanapi
+
+    :expectedresults: Host is provisioned succesfully.
+
+    :CaseAutomation: Automated
+    """
+    sat = module_provisioning_sat.sat
+    hostname = gen_string('alpha').lower()
+    module_vmware_hostgroup.group_parameters_attributes = [
+        {'name': 'package_upgrade', 'value': 'false', 'parameter_type': 'boolean'}
+    ]
```

Comment:
I would keep this as the host submission already takes a lot of time, this would eventually reduce the execution time and also would add the coverage on parameters.